### DISCUSSION TITLE
Should vibration work?

### DISCUSSION THREAD

**Discussion by Sergey5352** [nefarius/BthPS3#33] [Support]
Hello. I installed this driver for the PS3 joystick, went into the game and the vibration does not work for me. Is such a function implemented? Or do I have some problems?

**Comment by nefarius** [comment]
Vibration has nothing to do with this solution, it's the job of DsHidMini. Since you provided no details about your setup I can only guess your problem and recommend you read the extensive docs on https://vigem.org

Cheers

**Comment by Busteray** [comment]
Changing the HID mode to Xinput fixed that for me. SXS(steam) mode doesn't support rumble afaik.
